### ISSUE TITLE
Guard 2.11.0 causing errors when guard isn't used

### ISSUE THREAD

**Issue by danielwestendorf** [guard/guard#715]
Running tests on my ruby projects with guard 2.11.0 results in 100% passing tests. However, when I run `bundle exec rake test` I get the following output, causing my builds to fail. 
```
guard-2.11.0/lib/guard/ui.rb:26:in `logger': uninitialized constant Module::Lumberjack (NameError)
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/guard-2.11.0/lib/guard/ui.rb:193:in `block in _filtered_logger_message'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/guard-2.11.0/lib/guard/ui.rb:184:in `_filter'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/guard-2.11.0/lib/guard/ui.rb:191:in `_filtered_logger_message'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/guard-2.11.0/lib/guard/ui.rb:120:in `debug'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/notiffany-0.0.2/lib/notiffany/notifier/detected.rb:66:in `rescue in block (2 levels) in detect'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/notiffany-0.0.2/lib/notiffany/notifier/detected.rb:62:in `block (2 levels) in detect'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/notiffany-0.0.2/lib/notiffany/notifier/detected.rb:61:in `each'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/notiffany-0.0.2/lib/notiffany/notifier/detected.rb:61:in `detect'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/notiffany-0.0.2/lib/notiffany/notifier/detected.rb:61:in `block in detect'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/notiffany-0.0.2/lib/notiffany/notifier/detected.rb:60:in `each'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/notiffany-0.0.2/lib/notiffany/notifier/detected.rb:60:in `detect'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/notiffany-0.0.2/lib/notiffany/notifier.rb:105:in `initialize'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/notiffany-0.0.2/lib/notiffany/notifier.rb:42:in `new'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/notiffany-0.0.2/lib/notiffany/notifier.rb:42:in `connect'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/guard-2.11.0/lib/guard/notifier.rb:11:in `connect'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/guard-2.11.0/lib/guard/notifier.rb:30:in `notify'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/guard-minitest-2.3.2/lib/guard/minitest/notifier.rb:34:in `notify'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/guard-minitest-2.3.2/lib/guard/minitest/reporter.rb:11:in `report'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/minitest-5.5.0/lib/minitest.rb:627:in `each'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/minitest-5.5.0/lib/minitest.rb:627:in `report'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/minitest-5.5.0/lib/minitest.rb:129:in `run'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/minitest-5.5.0/lib/minitest.rb:56:in `block in autorun'
```

Downgrading to guard 2.10.0 results in no error output.

**Comment by e2** [comment]
Thanks for reporting this!

Could you create a tiny repository reproducing the problem?

That way I can:
- create a Cucumber feature test for this (https://github.com/guard/guard/tree/master/features)
- work out how to setup logging without creating unnecessary dependencies for upcoming work

Thanks!

**Comment by danielwestendorf** [comment]
Here is a repo with the problem. https://github.com/danielwestendorf/guard-error

**Comment by e2** [comment]
Thanks! I really appreciate the example repo, since it saved me a lot of time setting things up and testing!

**Comment by danielwestendorf** [comment]
No problem, thanks for the quick patch!

**Comment by e2** [comment]
It's released as Guard 2.11.1 ;) Thanks!
